### PR TITLE
mention prefix filtering 

### PR DESCRIPTION
**Pull request by TopiSenpai**

### CODE REVIEW

File: src/main/java/de/anteiku/kittybot/commands/utilities/KickCommand.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			return;
+		}
+		var members = ctx.getMentionedMembers();
+		if(members.size() == 0){
+			sendError(ctx, "Please mention a user");
+			return;
+		}
+		var user = ctx.getUser();
+		for(var member : members){
+			member.kick("").reason("Kicked by command from '" + user.getAsTag() + "'(" + user.getId() + ")").queue();
```
Comment (caneleex) [inline src/main/java/de/anteiku/kittybot/commands/utilities/KickCommand.java:41]: why not just `member.kick(reason)` here?

Comment (TopiSenpai) [reply src/main/java/de/anteiku/kittybot/commands/utilities/KickCommand.java:41]: this is wip I just fucked up not doing a draft :)
and the reason is for the audit log

File: src/main/java/de/anteiku/kittybot/commands/utilities/BanCommand.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			sendError(ctx, "You don't have enough permission to ban members");
+			return;
+		}
+		var members = ctx.getMentionedMembers();
+		if(members.isEmpty()){
+			sendError(ctx, "Please mention at least one user");
+			return;
+		}
+		for(var member : members){
+			member.ban(0, "").reason("Command  ran by '" + ctx.getUser().getAsTag() + "'(" + ctx.getUser().getId() + ")").queue();
```
Comment (caneleex) [inline src/main/java/de/anteiku/kittybot/commands/utilities/BanCommand.java:33]:
```suggestion
			member.ban(0, "").reason("Command ran by '" + ctx.getUser().getAsTag() + "'(" + ctx.getUser().getId() + ")").queue();
```

**Review by TopiSenpai — COMMENTED** [review]
lgtm to me
and I want to merge that now :p

**Pull request merged**